File: Python/POO2/vehiculo/Bicicleta.py
from vehiculo.Vehiculo import Vehiculo


class Bicicleta(Vehiculo):

    def __init__(self, velocidades):
        super().__init__()
        self.__velocidades = velocidades

    def getVelocidades(self):
        return self.__velocidades

    def setVelocidades(self, velocidades):
        self.__velocidades = velocidades

    def hacerCaballito(self):
        print("\nEstoy haciendo el caballito\n")

File: Java/POO2/vehiculo/Bicicleta.java
package vehiculo;

/**
 * Clase Bicicleta hereda de Vehiculo y su atributo y metodo propio
 * @author Juan Manuel Gonzalez Profumo
 * @version 1.0
 */
public class Bicicleta extends Vehiculo {
	private int velocidades;

	public Bicicleta(int velocidades) {
		super();
		this.velocidades = velocidades;
	}

	public int getVelocidades() {
		return velocidades;
	}

	public void setVelocidades(int velocidades) {
		this.velocidades = velocidades;
	}
	
	void hacerCaballido() {
		System.out.println("\nEstoy haciendo el caballito\n");
	}
}
